launcher/sdw-notify.py

```diff
@@ -4,11 +4,10 @@
 for too long without checking for security updates. Writes output to a logfile,
 not stdout. All settings are in Notify utility module.
 """
-
 import sys
 
-from sdw_notify import Notify
-from sdw_updater_gui import Updater
+from sdw_notify import Notify, strings
```

Comment:
I really like the separate module of strings.

If we were to extract the dialog below, then those strings could be encapsulated within that namespace :slightly_smiling_face: 

Not sure if that's the most idiomatic approach, but when thinking of reducing the API surface, I think:

```sh
tree .
 .
└── security_check         # or whatever name makes sense, it's the one that will show up in imports
           ├── __init__    # exposes the dialog class
           ├── prompt.py   # or whatever is a good name for the dialog class
           └── strings.py  # imported by the file above but not exposed by __init__
``` 

Usage could look like (just examples, I picked names to illustrate what I'd try to expose from the module):

```py
import security_check
user_choice = security_check.Prompt(sdapp_halted=True).exec()
# I personally like this best, because there is no namespace collapse, "Prompt" is always namspaced.
```

or

```py
from security_check import SecurityCheckPrompt
user_choice = SecurityCheckPrompt(sdapp_halted=True).exec()
# Might be more idiomatic? I don't like too much that "SecurityCheckPrompt" suddenly is exported by this module too - :grimacing: 
```

Reply:
@gonzalo-bulnes see if the changes in ed8d97f work for you in terms of separation of concerns :)

I _was_ hoping that I could completely eliminate the dependency on Qt components in `sdw-notify.py`, which I think would be very clean, but that requires just a little more time and thought to avoid issues with creating/handling `QApplication` (managing event loops). My goal would be to start here, perhaps make some similar improvements to the `sdw-launcher` in terms of improving the way it does its imports, and eventually move away from dependencies on Qt components in all but the view components. But I'd like to hear your thoughts.